### DISCUSSION TITLE
Error: Invalid request to Oauth API

### DISCUSSION THREAD

**Discussion by 0-BlackSpectrum-0** [Docile-Alligator/Infinity-For-Reddit#1561] [Q&A]
I'm trying to login in v6.3.0 and it shows this, I downloaded the app from github, anyway to fix this, sorry if it's already posted, if you can then please either link me tell me the solution or link me to the post that does 🙏
Thanks!

**Comment by FabianMPunkt** [comment]
Same issue here. Working fine with other clients.

**Comment by MonsieurBibo** [comment]
It's because of the new system that reddit uses. Now alternative clients need to pay high fees to reddit for API calls or otherwise the app is blocked. Try updating your app with the version on github (for exemple 6.3.2) and you will see for yourself.
